fem/bilininteg.cpp

```diff
@@ -2617,6 +2633,14 @@ void DGTraceIntegrator::AssembleFaceMatrix(const FiniteElement &el1,
    }
 }
 
+
+const IntegrationRule &DGTraceIntegrator::GetRule(
+   Geometry::Type geom, int order, FaceElementTransformations &T)
+{
+   int int_order = T.Elem1->OrderW() + 2*order;
```

Comment:
This is inconsistent with the formula in `AssembleFaceMatrix`. What was the reason to add `T.Elem1->OrderW()`?